# msg/Move.msg
uint32 object_id
uint32 target_id